js/src/dropdown.js

```diff
@@ -402,11 +400,19 @@ class Dropdown extends BaseComponent {
         continue
       }
 
-      if (event && ((event.type === 'click' &&
-          /input|textarea/i.test(event.target.tagName)) ||
-          (event.type === 'keyup' && event.key === TAB_KEY)) &&
-          dropdownMenu.contains(event.target)) {
-        continue
+      if (event) {
+        const dropdownForm = dropdownMenu.querySelector('form')
```

Comment:
Should we use SelectorEngine.findOne() here?

Reply:
That is better :+1:

Reply:
That is better :+1: